packages/amplify-provider-awscloudformation/src/configuration-manager.ts

```diff
@@ -94,13 +94,14 @@ export async function configure(context: $TSContext) {
 async function enableServerlessContainers(context: $TSContext) {
   const frontend = context.exeInfo.projectConfig.frontend;
   const { config = {} } = context.exeInfo.projectConfig[frontend] || {};
-  // TODO: check for headless mode parameter to avoid the question
-  const { ServerlessContainers } = await prompt({
-    type: 'confirm',
-    name: 'ServerlessContainers',
-    message: 'Do you want to enable container-based deployments?',
-    default: config.ServerlessContainers === true,
-  });
+  const { ServerlessContainers } =
+    config.ServerlessContainers ??
```

Comment:
Wouldn't this prevent changing your answer after the first time it's run?

Reply:
Not according to my tests - If `"ServerlessContainers:value" is present in the passed frontend config, it'll be used instead of the default value.

Reply:
Well, it turns out I had the de-structuring wrong anyway, hah (or maybe this is what you were pointing out?) It should be:
```javascript
const serverlessContainers =
    config.ServerlessContainers ??
    (await prompt({
      type: 'confirm',
      name: 'ServerlessContainers',
      message: 'Do you want to enable container-based deployments?',
      default: config.ServerlessContainers === true,
    })).ServerlessContainers;
```
I'll push this fix up with the unit test

Reply:
What I meant is that if I run `amplify-dev configure project` and select the Container-based deployments option, I can select if I want to enable it or not. If I run the same command again, I no longer have the option to enable/disable it. The problem is that `config.ServerlessContainers ??` only lets the prompt run if `ServerlessContainers` is `undefined`, which is only the case on the first run.

Reply:
I see now, that makes sense. Thanks for the detailed explanation